bin/impl/setup.sh

```diff
@@ -34,6 +34,13 @@ verify_exist_hash $HADOOP_TARBALL $HADOOP_MD5
 verify_exist_hash $ZOOKEEPER_TARBALL $ZOOKEEPER_MD5
 verify_exist_hash $SPARK_TARBALL $SPARK_MD5
 
+nslookup `hostname` &> /dev/null
```

Comment:
http://blog.smalleycreative.com/linux/nslookup-is-dead-long-live-dig-and-host/

I am uncertain if we should be concerned about nslookup being deprecated.